tools/linkchecker-docker/neurodocker-build.sh

```diff
@@ -9,19 +9,19 @@
 #     for instructions
 
 # tag for patched linkchecker -- will also serve our image version
-version="9.4.0.anchorfix1"
+version="v10.0.1"
```

Comment:
Just FTR: properly (?) fixed in https://github.com/linkchecker/linkchecker/pull/462 (issue https://github.com/linkchecker/linkchecker/issues/460) which was `v10.0.0~14^2` (thus this version above includes it) instead of my adhoc https://github.com/linkchecker/linkchecker/pull/194 (which I believe was what that version pointed to).